fmriprep/workflows/bold/t2s.py

```diff
@@ -109,7 +116,7 @@ def init_bold_t2s_wf(
     dilate_mask = pe.Node(BinaryDilation(radius=2), name='dilate_mask')
 
     t2smap_node = pe.Node(
-        T2SMap(echo_times=list(echo_times)),
+        T2SMap(echo_times=list(echo_times), fittype=config.workflow.me_t2s_fit_method),
         name='t2smap_node',
         mem_gb=2.5 * mem_gb * len(echo_times),
```

Comment:
reading that curvefit is more memory intensive, will this estimate need to be adjusted?

Reply:
The estimate there came from #2728, wherein I profiled t2smap with curvefit, so I think it should be accurate for curvefit. Would having mem_gb depend on the parameter (e.g., have a lower value for loglin) be a good idea?

Reply:
ah, if we're covering the upper bound then no need to get in the weeds.